exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -1,27 +1,248 @@
-OpenTelemetry Prometheus Remote Write Exporter
-==============================================
+OpenTelemetry Python SDK Prometheus Remote Write Exporter
```

Comment:
Not sure if we need the "Python SDK" part (not consistent with other exporters as well)